.travis.yml

```diff
@@ -2,43 +2,59 @@ language: node_js
 node_js: 
   - "8"
   - "10"
+  - "12"
 
 script: npm run unit-test
 
+common: &integration-common
+  script:
+    - npm run build
+    - node build_release_package_json.js
+    - npm run integration-test
+  node_js: "8"
```

Comment:
I think you can remove the definition of node_js in common. It can be defined for every step